kontrakter/politi/varetekt/1.0/begjaeringVaretekt.schema.json

```diff
@@ -119,19 +117,592 @@
+		  },
+		  "additionalProperties": false
+	  },
+    "land": {
+      "description": "ISO-3166, Kosovo kommer",
+      "type": "object",
+      "properties": {
+        "kode" : {"type": "string", "pattern": "[A-Z]+", "minLength": 3, "maxLength": 3},
+        "navn" : {"type": "string"}
+        },
+      "required" : ["kode"]
+    },
+    "organisasjon": {
+      "description": "Organisasjon som kan sende eller motta JustisHub meldinger",
+      "type": "object",
+      "properties": {
+        "organisasjonsnummer": {"type": "string"},
+        "navn": {"type": "string"},
+        "kontakt" : {"$ref" : "#/definitions/kontaktInfo"}
+        },
+      "required": ["organisasjonsnummer","navn"],
+      "additionalProperties": false
+    },
+	  "ansatt": {
+      "description": "Saksbehandler, jurist etterforsker hos politiet, ansatte hos domstolene eller kriminalomsorgen",
+      "type": "object",
+      "properties": {
+        "brukeridentifikasjon": {"type": "string"},
+        "tittel" :  {"type": "string"},
+        "navn": {"$ref" : "#/definitions/personnavn"},
+        "kontakt" : {"$ref" : "#/definitions/kontaktInfo"}
+      },
+      "required" : ["navn"],
+      "additionalProperties": false
+    },
+
+  "forsendelse" : {
+		  "properties": {
+			"meldingsId" : {"type": "string"},
+			"sendtTid" : {"type": "string", "format" : "date-time"},
+			"avsender" : {"$ref": "#/definitions/avsender"},
+			"mottaker" : {"$ref": "#/definitions/mottaker"}
+		  },
+		  "required" : ["meldingsId", "avsender", "mottaker", "sendtTid"],
+		  "additionalProperties": false
+	  },
+	  "avsender" : {
+		  "properties": {
+			  "organisasjon" : {"$ref": "#/definitions/organisasjon"},
+			  "saksbehandler" : {"$ref": "#/definitions/ansatt"}
+		  },
+		  "required" : ["organisasjon"],
+		  "additionalProperties": false
+    },
+	  "mottaker" : {
+		  "properties": {
+			  "organisasjon" : {"$ref": "#/definitions/organisasjon"}
+		  },
+		  "required" : ["organisasjon"],
+		  "additionalProperties": false
+    },
+		"anke": {
+      "description": "Anke av fengsling/restriksjon/kjennelse",
+      "type": "object",
+      "required": ["anketAv", "oppsettendeVirkning"],
+      "additionalProperties": false,
+			"properties": {				
+				
+				"anketDato": {"type": "string", "format": "date"},
+				"anketAv": {"$ref": "#/definitions/anketAvType"},
+				"oppsettendeVirkning": {"type": "boolean"					
+				}
+			}
+		},		
+		"anketAvType": {
+			"description": "Hvem av partene har anket",
+			"enum": ["SIKTET","FORSVARER", "AKTOR"],
+			"type": "string"
+		},
+
+    "avgjoerelseDomstol": {
+      "description": "Kjennelse(r) fra retten, etter anke kan antagelig flere retter være involvert og da må vi endre denne",
+      "type": "object",
+      "required": ["kjennelse"],
+      "additionalProperties": false,
+      "properties": {
+        "merknad": {"type": "string"},
+        "kjennelse": {"type": "string"}
+      }
+    },
+    "fengsling": {
```

Comment:
`#/definitions/fengsling` er definert to ganger. Tidligere på linje 84

Reply:
Sletter duplikat "fengsling"

Reply:
format date skal ikke brukes til Kriminalomsorgen, men til domstolene skal vi bruke det.